src/components/calcite-button/calcite-button.scss

```diff
@@ -11,13 +11,17 @@
   --calcite-button-dark: #{$blk-180};
   --calcite-button-dark-hover: #{$blk-170};
   --calcite-button-dark-press: #{$blk-190};
+  --calcite-button-transparent-hover: rgba(0, 0, 0, 0.05);
+  --calcite-button-transparent-press: rgba(0, 0, 0, 0.08);
 }
 
 // dark theme
 :host([theme="dark"]) {
   --calcite-button-dark: #{$blk-180};
```

Comment:
so we need to at some point take care of these vars and expand the actual theme to cover them. Currently these will not work when inheriting the theme from an ancestor because they expect a theme attribute to be set on the actual host, which it may not.

We should have names for these transparent vars in the theme like `calcite-ui-transparent-1`, `calcite-ui-transparent-2`, etc and then reference them here so that they work with the inherited theme values

Reply:
Roger - these are just "overlay filters" for a transparent button... which shouldn't actually affect anything that's themed? (ie, I don't think we'd want to allow people to set a "them" for a transparency filter - or am I wrong there?)

Reply:
Or do you mean dark variables - in that case yes, there's an open issue to resolve that whole dark theme / dark button and light theme / light button deal https://github.com/Esri/calcite-components/issues/576

Reply:
no, you're not wrong. We don't necessarily want to allow overriding them, but we do need them to flip in light/dark further up the tree. We could add a lookup for theme or something if the button is transparent and key off a class or something. Ie:

```
let transparentClass = "";
if (this.transparent) {
  this.transparentClass = getElementTheme()
}
```

And then do  `:host([transparent].dark) { }` or something maybe.

Reply:
Gotcha - yeah, we also have "overlay" variables for the progress bar in auto-dismiss alerts we could use something for (transparent-overlay-1/2) - https://github.com/Esri/calcite-components/blob/65d86d9fc41fcf9b8e4310233b81ce27cdf24a07/src/components/calcite-alert/calcite-alert.scss#L3-L10